code/CodeTester.cpp

```diff
@@ -2103,27 +2102,13 @@ int main(
         }
     }
     else
-    {
-        if (strncmp(argv[1], "Static", 6) == 0)
-        {
-            XMLEndpointParser parser;
-            std::string file = argv[1];
+    {        
+        XMLProfileManager parser;
```

Comment:
We might as well move this test over to dds-doctest as well

Reply:
ok